third_party/musl/src/string/memchr.c

```diff
@@ -8,6 +8,9 @@
 #define HIGHS (ONES * (UCHAR_MAX/2+1))
 #define HASZERO(x) ((x)-ONES & ~(x) & HIGHS)
 
+#ifdef __GNUC__
+__attribute__((no_sanitize("address")))
```

Comment:
Not sure if this applies but since we are making changes to  third_party code , does this come under a flag like USE_COBALT_CUSTOMIZATIONS or maybe there's  another flag i dont know of.

Reply:
Updated